src/types.ts

```diff
@@ -179,3 +179,47 @@ export interface RawTechnicalQuestion {
   follow_ups: string;
   source: string;
 }
+
+export type ReportType = "interview" | "question";
+
+export type ReportStatus = "pending" | "resolving" | "completed";
+
+export interface RawReportBody {
+  associated_id: string;
+  reporter_user_id: number;
+  type: ReportType;
+  reason: string;
+}
+
+export interface ReportBody {
+  associatedId: string;
+  reporterUserId: number;
+  type: ReportType;
+  reason: string;
+}
+
+export interface RawReport {
+  admin_id: null | string;
+  admin_notes: null | string;
+  associated_id: null | string;
+  created: string;
+  reason: string;
+  report_id: string;
+  reporter_user_id: number;
+  status: ReportStatus;
+  type: ReportType;
+  updated: string;
+}
+
+export interface Report {
+  adminId?: string;
+  adminNotes?: string;
+  associatedId?: string;
```

Comment:
what is associatedId here? does it make sense to have a report without one?

Reply:
yes as feedback